smart_open/tests/test_smart_open.py

```diff
@@ -886,21 +947,18 @@ class CompressionFormatTest(unittest.TestCase):
     Test that compression
     """
 
-    TEXT = 'Hello'
-
     def write_read_assertion(self, suffix):
         with tempfile.NamedTemporaryFile('wb', suffix=suffix, delete=False) as infile:
             test_file = infile.name
 
-        text = self.TEXT.encode('utf8')
         with smart_open.smart_open(test_file, 'wb') as fout:  # 'b' for binary, needed on Windows
-            fout.write(text)
+            fout.write(SAMPLE_BYTES)
 
         with open(test_file, 'rb') as fin:
-            self.assertNotEqual(text, fin.read())
+            self.assertNotEqual(SAMPLE_BYTES, fin.read())
 
         with smart_open.smart_open(test_file, 'rb') as fin:
-            self.assertEqual(fin.read().decode('utf8'), self.TEXT)
+            self.assertEqual(fin.read().decode('utf8'), SAMPLE_TEXT)
```

Comment:
Isn't this the same as:

```suggestion
            self.assertEqual(fin.read(), SAMPLE_BYTES)
```